scripts/annotations/CSVMerger.java

```diff
@@ -0,0 +1,94 @@
+package annotations;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+
+import static annotations.CSVUtils.*;
+
+/**
+ * Merges several CSV files into a single CSV file. First line of the CSV files
+ * must be a header line! Ignores entries starting with a '#' (also whole
+ * columns if their header starts with a '#'!)
+ */
+public class CSVMerger {
+
+    // BEGIN Parameters
+    
+    static final String[] files = {
+            "/Users/dlindner/idr0040/idr0000-experimentB-assays_DoseResponsesSupFigs.csv",
```

Comment:
Should these files be moved under revision control and/or the paths adjusted to point to `/uod/idr/filesets`?